src/lib.rs

```diff
@@ -381,7 +381,7 @@ Theere are 2 ways to support multiple configurations at the same time.
 Just split client and server logic. Then for listen server and singleplayer run both the server and client,
 just don't accept outside connections for singleplayer.
 
-However, **running the client and server in a single app is not supported**. We rely on change detection to
+However, **running the client and server connected to each other won't work**. We rely on change detection to
```

Comment:
This looks more confusing than the original. What are you trying to say here?

Reply:
Never mind, reverted this change :slightly_smiling_face: